src/components/Summary/AlterationSummary.vue

```diff
@@ -232,6 +238,23 @@ export default class AlterationSummary extends Mixins(
   /** Prop to perform validation. */
   @Prop() readonly validate: boolean
 
+  /** The future effective fee for Alteration. */
+  futureEffectiveFees: number = 0
+
+  async mounted () {
+    // fetches the fee prices to display in the text
+    const feePrices = await this.fetchFeePrices()
+    this.futureEffectiveFees = feePrices.futureEffectiveFees
+  }
+
+  /** Fetches the Fee prices to display in the text */
+  private async fetchFeePrices (): Promise<FeesIF> {
+    const result = await Promise.resolve(this.fetchFilingFees(FilingCodes.ALTERATION,
+      this.getEntityType, true))
```

Comment:
Minor structural thing here. 
I'm not sure about other reviewers but if they didn't all fit in one line, it's nice to break the parameters into their own lines respectively. 
ie `Promise.resolve(
this.fetchFilingFees(FilingCodes.ALTERATION,this.getEntityType, true)
)`

Reply:
I think you don't need `Promise.resolve` here (or elsewhere you do this).

`Await` takes care of resolving the promise.

PS If you need to catch the promise being rejected, it's like this:
`const foo = await bar().catch(() => doSomething())`

Reply:
Ok. Yeah, I changed that. Thanks.